.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -26,6 +26,22 @@ body:
   validations:
     required: true
 
+- type: input
+  attributes:
+    label: Helm chart version
+    description: Helm chart version that you encounterd problem.
+    placeholder: ex. 0.16.2
```

```suggestion
    placeholder: E.g. 0.16.2
```

I am not an english expert but I think "ex" is an abbreviation for exercise.

---

Thank you for reviewing!
I fixed on https://github.com/argoproj/argo-helm/pull/1318/commits/3aea2fc6e15ca7036c46cbdd43e76eeba894c357 (exiting formats were all lowercase, so I followed them) . 🙋